src/main/resources/application.properties

```diff
@@ -53,38 +49,38 @@ spring.data.mongodb.database: eiffel_intelligence
 # spring.data.mongodb.username:
 # spring.data.mongodb.password:
 # Configure the session storage expiration time for logged in users of Eiffel Intelligence front-end
-server.session-timeout: 1200
+server.session.timeout: 1200
 # Configure the database collection name to store sessions
 sessions.collection.name: sessions
 
 # Name of the database collection where aggregated objects are stored
-aggregated.collection.name: aggregated_objects
+aggregations.collection.name: aggregations
 # Configure time-to-live value (seconds) for aggregated objects. This needs to be set if
 # using the example rule set 'AllEventsRules' to avoid the storage growing too large.
 # Recommended setting is 10 minutes.
-aggregated.collection.ttlValue:
+aggregations.collection.ttl:
 # The name of the database collection where to store the mapping between
 # aggregated objects and id of the Eiffel events that contributed to those
-event_object_map.collection.name: event_object_map
+event.object.map.collection.name: event_object_map
 # Name of the database collection where subscriptions are stored
-subscription.collection.name: subscription
+subscriptions.collection.name: subscriptions
 # Name of the database collection where we store pair of 'subscription:aggregated object'
 # where notification has been sent and should not be repeated
-subscription.collection.repeatFlagHandlerName: subscription_repeat_handler
+subscriptions.repeat.handler.collection.name: subscriptions_repeat_handler
 # The waitlist contains Eiffel events which are waiting to be aggregated
-# Configure the database collection name for the waitlist
+# Configure the database collection name for the wait list
 waitlist.collection.name: wait_list
 # Set the time-to-live value for Eiffel events in the waitlist database (seconds)
-waitlist.collection.ttlValue: 600
+waitlist.collection.ttl: 600
 # Resend information for unprocessed events in wait list, time in milliseconds
-waitlist.initialDelayResend: 2000
-waitlist.fixedRateResend: 15000
+waitlist.resend.initial.delay: 2000
+waitlist.resend.fixed.rate: 15000
```

Comment:
is fixed.state specific enough? Perhaps interval might be better? Because I assume this is how often we resend events?

Reply:
I think it's named like this because of the threadpool. These properties are used by the spring like this: @Scheduled(initialDelayString = "${waitlist.resend.initial.delay}", fixedRateString = "${waitlist.resend.fixed.rate}")
For clarity our property could be named waitlist.resend.interval though.

Reply:
No it's fine to keep them as is then. I guess it's easier to follow Springs documentation for a full explanation of what the properties are used for if neccesary.